### PR TITLE
be more explicit about the privacy of id channels

### PR DESCRIPTION
**Pull request by zkat**

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Examples of unacceptable behavior by participants include:
 * Publishing others' private information, such as a physical or electronic
   address, without explicit permission. This includes any sort of "outing" of
   of any aspect of someone's identity without their consent.
+* Publishing screenshots or quotes, specially from identity channels, without a user's explicit consent.
```
Comment (aredridel) [inline code-of-conduct.md]: s/specially/especially/

Comment (aredridel) [inline code-of-conduct.md]: I like the emphasis, but as a rule doesn't actually make a distinction: for channels, don't. For ID channels: don't

Comment (zkat) [reply code-of-conduct.md]: the intention behind that emphasis is to clarify that enforcement will probably be harsher/more aggressive for those comms. Do you have a better wording that you'd suggest? I had trouble constructing this clause.

Comment (aredridel) [reply code-of-conduct.md]: That makes sense in that context.

**Comment by cjwilburn** [comment]
👍

**Pull request merged**